### PR TITLE
Respond with 404 when the source image can not be found in OpenStack Object Storage

### PR DESCRIPTION
**Pull request by joec4i**
Does what the title says so imgproxy doesn't throw a 500 when it should return 404.

Review please. 🙏

### CODE REVIEW

File: transport/swift/swift.go [hunk 1]
```diff
@@ -46,12 +47,25 @@ func (t transport) RoundTrip(req *http.Request) (resp *http.Response, err error)
...
+
 	if err != nil {
+		if errors.Is(err, swift.ObjectNotFound) || errors.Is(err, swift.ContainerNotFound) {
+			return &http.Response{
+				StatusCode: http.StatusNotFound,
+				Proto:      "HTTP/1.0",
+				ProtoMajor: 1,
+				ProtoMinor: 0,
+				Header:     header,
+				Body:       nil,
```
Comment (DarthSim) [inline transport/swift/swift.go:60]: I'd add a body with the error message:

```suggestion
				Body:       io.NopCloser(strings.NewReader(err.Error())),
```

**Review by DarthSim — CHANGES_REQUESTED** [review]
Hi @joec4i!

Everything looks good! Let's add a response body and I'll merge this.

File: transport/swift/swift.go (hunk 1 above)
Comment (joec4i) [reply transport/swift/swift.go]: Thanks for the quick review. It's updated now. Thanks @DarthSim

**Comment by DarthSim** [comment]
Thanks for your contribution!

**Pull request merged**